Make script executable again

jsf9k opened the pull request:
## 🗣 Description ##

This pull request makes a script executable again, fixing a bug introduced in #148.

## 💭 Motivation and context ##

[cisagov/pca-gophish-composition](https://github.com/cisagov/pca-gophish-composition) cannot pass its tests without this change.

Note that I invested very little effort into resolving the `shellcheck` errors in the script.  This is because as soon as cisagov/pca-gophish-composition#62 is merged I intend to remove cisagov/gophish-tools from it.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

jsf9k committed:
Make getenv executable again

It was made non-executable in #148, but
cisagov/pca-gophish-composition does not pass its tests unless the
script is executable.

jsf9k committed:
Fix some shellcheck warnings

jsf9k committed:
Ignore some shellcheck errors

This repo is not long for this world, so there is no sense in
investing a lot of time into resolving these errors.

jsf9k committed:
Bump project version from 1.0.0 to 1.0.1

mcdonnnj reviewed (changes requested):
Please remove the [instruction in the Dockerfile that makes `getenv` executable](https://github.com/cisagov/gophish-tools/blob/548d4886499fe764beb3e07d968470aca758a9c9/Dockerfile#L34). The key problem is the loss of the shebang in #148 because the file has remained executable.

Not specifically related but the Docker Hub web UI is giving me wonky results where the `latest` tag it lists is definitely not the most recent image you get when you pull the `latest` tag. I was trying to cross reference things and was confused on why the image I pulled said it was made days ago but the web UI said the `latest` tag was last pushed over two years ago. Just something to watch out for if you're looking things up in the future.

jsf9k committed:
Remove commend to make getenv eecutable

This is no longer needed since I made the script executable in the
repository.

Co-authored-by: Nick <50747025+mcdonnnj@users.noreply.github.com>

jsf9k commented:
> Please remove the [instruction in the Dockerfile that makes `getenv` executable](https://github.com/cisagov/gophish-tools/blob/548d4886499fe764beb3e07d968470aca758a9c9/Dockerfile#L34). The key problem is the loss of the shebang in #148 because the file has remained executable.

Please see commit 58e6036.

jsf9k commented:
> Not specifically related but the Docker Hub web UI is giving me wonky results where the `latest` tag it lists is definitely not the most recent image you get when you pull the `latest` tag. I was trying to cross reference things and was confused on why the image I pulled said it was made days ago but the web UI said the `latest` tag was last pushed over two years ago. Just something to watch out for if you're looking things up in the future.

Got it.  This repo isn't long for this world, though, so I won't worry about that.
